### DISCUSSION TITLE
rsync ability in SFTPGo

### DISCUSSION THREAD

**Discussion by krushnadattaswain** [drakkan/sftpgo#525] [Q&A]
Hi @drakkan , I saw rsync support in SFTPGo. Do you have any document supporting rsync.

**Comment by drakkan** [comment]
Hi,

please take a look [here](https://github.com/drakkan/sftpgo/blob/main/docs/ssh-commands.md), we just use the rsync system command

**Reply by ciupicri** [reply]
[updated link](https://sftpgo.github.io/latest/ssh/#ssh-commands)
